src/compiler/scala/tools/asm/LabelAccess.java

```diff
@@ -0,0 +1,18 @@
+package scala.tools.asm;
+
+/**
+ * Temporary class to allow access to the package-private status field of class Label.
+ */
+public class LabelAccess {
```

Comment:
Should I do a new asm release with the `status` made public before we merge this PR?

Reply:
I don't really mind if we merge this first.